dunegdt/docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM pymor/python_PYVER:dev
+
+MAINTAINER René Fritze <rene.fritze@wwu.de>
+
+RUN apt-get update && \
+    apt-get -y install cmake git g++ \
+    openmpi-bin libopenmpi-dev \
+    bison build-essential cmake flex gfortran libboost-system-dev \
+                libboost-thread-dev libboost-filesystem-dev libboost-date-time-dev \
+                libboost-timer-dev libboost-chrono-dev libsuperlu-dev libtool \
+                pkg-config libopenblas-dev libtbb-dev libeigen3-dev && \
+    pip install mpi4py
+ENV DUNE_GDT_VERSION=master \
+  DUNE_SRC_DIR=/src \
+  DUNE_BUILD_DIR=/src/build
+
+RUN git clone https://zivgitlab.uni-muenster.de/ag-ohlberger/dune-community/dune-gdt-super ${DUNE_SRC_DIR} && \
+    cd ${DUNE_SRC_DIR} && \
+    git checkout ${DUNE_GDT_VERSION} && \
+    git rm -rf dune-uggrid dune-testtools && \
+    cd ${DUNE_SRC_DIR}  && \
+    git submodule update --init --recursive
```

Comment:
does this restore testtools and uggrid?

Reply:
Nope. It would if line 20 was a plain `rm`, but since it's a `git` command it actually removes the submodules. Dunecontrol would also fail with testtools present since I'm disallowing venv creation in opts.